integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/CatalogsPageTest.java

```diff
@@ -35,13 +35,15 @@
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.MethodOrderer;
 import org.junit.jupiter.api.Order;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestMethodOrder;
 import org.openqa.selenium.By;
 
+@Disabled
```

Comment:
The `CatalogPageTest` shouldn't rely on Trino, Can you decouple them?

Reply:
It's Depend on the Trino docker environment

Reply:
But it doesn't use Trino, right?

Reply:
Yes